cmd/src/users_verify.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"text/template"
+)
+
+const SubCommand = "verify-emails"
+
+// input = `[]VerifyEmailJson`
```

Comment:
?

Reply:
it was designed to document what `.` represents in the `text/template`, but if you want that huge blob of text inline in the `func` (:cry:) then I guess the documentation need falls away